org.thingpedia.iot.speaker/manifest.tt

```diff
@@ -0,0 +1,61 @@
+abstract class @org.thingpedia.iot.speaker
+#_[thingpedia_name="Speaker"]
+#_[thingpedia_description="Interface for Speaker."]
+#_[description="Interface for Speaker."]
+#[license="CC-0"]
+#[license_gplcompatible=true]
+#[subcategory="home"]
+{
+  monitorable query state(out state : Enum(on,off))
+  #_[canonical="switch state"]
+  #_[confirmation="the state of $__device"]
+  #_[formatted=["It is ${state}"]];
```

Comment:
What's the definition of speaker state in Home Assistant? With Bluetooth, you don't know on/off, all you know is connected/disconnected.

Reply:
Changed state() to power(). HA device can be queried about whether it is on/off. This throws an error for Bluetooth.